concepticondata/conceptlists/Seifart-2015-414.tsv

```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-114	114	spider	843	SPIDER	147
+Seifart-2015-414-115	115	jigger (tunga penetrans)	438	CHIGGOE (JIGGER FLEA)	148
+Seifart-2015-414-116	116	bush	1683	BUSH	149
+Seifart-2015-414-117	117	open grassland	260	SAVANNA	150
+Seifart-2015-414-118	118	hill	614	HILL	151
+Seifart-2015-414-119	119	tree	906	TREE	152
+Seifart-2015-414-120	120	leaf	628	LEAF	153
+Seifart-2015-414-121	121	tree leaf	628	LEAF	154
+Seifart-2015-414-122	122	root	670	ROOT	158
+Seifart-2015-414-123	123	seed	714	SEED	159
+Seifart-2015-414-124	124	stick	1295	STICK	160
+Seifart-2015-414-125	125	grass	606	GRASS	161
+Seifart-2015-414-126	126	cassava (bread)	925	CASSAVA	163
+Seifart-2015-414-127	127	tobacco	974	TOBACCO	165
+Seifart-2015-414-128	128	cotton	1850	COTTON	166
+Seifart-2015-414-129	129	gourd	411	GOURD	167
+Seifart-2015-414-130	130	yam	410	YAM	168
+Seifart-2015-414-131	131	annatto	349	ACHIOTE	170
+Seifart-2015-414-132	132	chili pepper	919	CHILI PEPPER	171
+Seifart-2015-414-133	133	coca	137	COCA	172
+Seifart-2015-414-134	134	banisterium	276	BANISTERIUM	173
+Seifart-2015-414-135	135	plantain	399	PLANTAIN	174
+Seifart-2015-414-136	136	palm	1181	PALM TREE	175
+Seifart-2015-414-137	137	salt	1274	SALT	179
+Seifart-2015-414-138	138	chicha (of peach palm fruits)	441	CHICHA	180
+Seifart-2015-414-139	139	one	1493	ONE	181
+Seifart-2015-414-140	140	two	1498	TWO	182
+Seifart-2015-414-141	141	five	493	FIVE	185
+Seifart-2015-414-142	142	first	728	FIRST	191
+Seifart-2015-414-143	143	last	1529	LAST (FINAL)	192
+Seifart-2015-414-144	144	drum	908	DRUM	194
+Seifart-2015-414-145	145	ear ornament			196
+Seifart-2015-414-146	146	mask	2005	MASK	197
+Seifart-2015-414-147	147	medicine man	394	SHAMAN (FOLK HEALER)	198
+Seifart-2015-414-148	148	chief	301	CHIEFTAIN	199
+Seifart-2015-414-149	149	i	1209	I	200
+Seifart-2015-414-150	150	thou	1215	THOU	201
+Seifart-2015-414-151	151	he	1211	HE	202
+Seifart-2015-414-152	152	she	1834	SHE	203
+Seifart-2015-414-153	153	it	169	IT	204
+Seifart-2015-414-154	154	we	1212	WE	205
+Seifart-2015-414-155	155	ye	1213	YOU	206
+Seifart-2015-414-156	156	they	817	THEY	207
+Seifart-2015-414-157	157	my hand			208
+Seifart-2015-414-158	158	thy hand			209
+Seifart-2015-414-159	159	his hand			210
+Seifart-2015-414-160	160	our hands			211
+Seifart-2015-414-161	161	your hands			212
+Seifart-2015-414-162	162	their hands			213
+Seifart-2015-414-163	163	my bow			214
+Seifart-2015-414-164	164	thy bow			215
+Seifart-2015-414-165	165	his bow			216
+Seifart-2015-414-166	166	our bow			217
+Seifart-2015-414-167	167	your bow			218
+Seifart-2015-414-168	168	their bow			219
+Seifart-2015-414-169	169	big	1202	BIG	220
+Seifart-2015-414-170	170	small	1246	SMALL	221
+Seifart-2015-414-171	171	cold	1287	COLD	222
+Seifart-2015-414-172	172	hot	1286	HOT	223
+Seifart-2015-414-173	173	good	1035	GOOD	224
+Seifart-2015-414-174	174	bad	1292	BAD	225
+Seifart-2015-414-175	175	white	1335	WHITE	226
+Seifart-2015-414-176	176	black	163	BLACK	227
+Seifart-2015-414-177	177	go!			228
```

Comment:
These cases are a bit strange and I noticed that some lists are mapped go! to GO and come! to COME. Since the verbs don't seem to appear anywhere else on the list, I'd map them, but could you open an issue where we discuss these cases?

Reply:
All of them do appear in an infinitive form in the list. But I can open an issue for us to discuss the cases where they don't.

Reply:
there is "come" in line 227. But "go" does seem to be missing indeed

Reply:
So, I'll map `go!` with [695 GO](https://concepticon.clld.org/parameters/695), leave the rest unmapped, and open an issue about these cases.

Reply:
There's already a response on #1337 on this topic from @LinguList.